### DISCUSSION TITLE
"Can FDS predict the suppression of a spray burner fire using a water mist system?

### DISCUSSION THREAD

**Discussion by aina47k** [firemodels/fds#12725] [Q&A]
Can FDS predict the suppression of a spray burner fire using a water mist system?
I would like to simulate the IMO 1165 test using FDS.
I have already run the ‘water_fuel_spray’ example. As a result, the suppression of the spray fire was not visible.
It would be great if someone could share information on this."

**Comment by mcgratta** [comment]
`water_fuel_spray` does not test suppression. Typically water mist works in a closed space by decreasing the oxygen supply via dilution by evaporated water vapor. FDS works that way as well. Look in the Validation Guide for examples of suppression by water mist, for example, USCG/HAI experiments.

**Reply by aina47k** [reply]
Thank you for providing valuable information! :)
